internal/reader/rmn_home.go

```diff
@@ -122,40 +121,24 @@ func (r *rmnHomePoller) poll() {
 }
 
 func (r *rmnHomePoller) fetchAndSetRmnHomeConfigs(ctx context.Context) error {
-	var versionedConfigWithDigests []VersionedConfigWithDigest
+	retVal := GetAllConfigsResponse{}
 	err := r.contractReader.GetLatestValue(
 		ctx,
 		r.rmnHomeBoundContract.ReadIdentifier(consts.MethodNameGetAllConfigs),
 		primitives.Unconfirmed,
 		map[string]interface{}{},
-		&versionedConfigWithDigests,
+		&retVal,
 	)
 	if err != nil {
 		return fmt.Errorf("error fetching RMNHomeConfig: %w", err)
 	}
-
-	if len(versionedConfigWithDigests) != expectedRMNHomeConfigs {
-		r.lggr.Warnw(
-			"unexpected number of RMNHomeConfigs",
-			"numConfigs", len(versionedConfigWithDigests),
-			"expected", expectedRMNHomeConfigs)
-		return fmt.Errorf("unexpected number of RMNHomeConfigs")
-	}
-
-	primaryConfigDigest := versionedConfigWithDigests[0].ConfigDigest
-	if primaryConfigDigest.IsEmpty() {
-		r.lggr.Debugw("primary config digest is empty")
-	}
-
-	secondaryConfigDigest := versionedConfigWithDigests[1].ConfigDigest
-	if secondaryConfigDigest.IsEmpty() {
-		r.lggr.Debugw("secondary config digest is empty")
-	}
+	r.lggr.Infof("Fetched RMNHomeConfigs: %v", retVal)
```

Comment:
Fields might be better here due to JSON encoding in the field logger

```suggestion
	r.lggr.Infow("Fetched RMNHomeConfigs", "rmnHomeConfigs", retVal)
```